aten/src/ATen/cpu/vec/vec256/vec256_int.h

```diff
@@ -888,6 +1156,12 @@ Vectorized<int8_t> inline operator*(const Vectorized<int8_t>& a, const Vectorize
   return int_elementwise_binary_256(a, b, std::multiplies<int8_t>());
 }
 
+template <>
+Vectorized<uint8_t> inline operator*(const Vectorized<uint8_t>& a, const Vectorized<uint8_t>& b) {
+  // We don't have an instruction for multiplying int8_t
```

Comment:
```suggestion
  // We don't have an instruction for multiplying uint8_t
```

Reply:
Same as above, thanks for pointing to this.